doc/operation-and-maintenance/Cluster-configuration-and-node-management.md

```diff
@@ -126,32 +106,37 @@ mongooseimctl leave_cluster
 
 It only makes sense to use it if the node is the part of a cluster, e.g `join_cluster` was called from that node before.
 
-The successful output from the above command starts with `You have successfully left the node from the cluster`.
+Similarly to `join_cluster` a warning and a question will be displayed unless option `-f` is added to the command.
+
+The successful output from the above command may look like the following:
+
+`You have successfully left the node mongooseim2@localhost from the cluster`.
+
+### Removing a node from the cluster
 
 To remove another node from the cluster, call the following command from one of the cluster members:
 
 ```bash
 mongooseimctl remove_from_cluster RemoteNodeName
 ```
 
-where `RemoteNodeName` is a name of the node that we'd like to remove from our cluster. 
+where `RemoteNodeName` is a name of the node that we'd like to remove from our cluster.
 This command could be useful when the node is dead and not responding and we'd like to remove it remotely.
-The successful output from above command starts with `The node has been removed from the cluster`
+The successful output from above command may look like the following:
 
-#### MongooseIM before 1.7.0
-
-For an older `MongooseIM` please see the previous versions of the documentation.
+`The node mongooseim2@localhost has been removed from the cluster`
 
 ### Cluster status
 
-To see if a newly added node is properly clustered or to see how the cluster looks like, the following commands can be useful (on any running node in the cluster).
+To see if a newly added node is properly clustered or to see how the cluster looks like,
```

Comment:
You can use the following commands on any of the running nodes to examine the cluster or to see if a newly added node is properly clustered:

Reply:
Brilliant!